user/details/style.css

```diff
@@ -278,3 +278,32 @@ footer p a {
     height: 20px;
   }
 }
+
+.accordion-disabled {
+  opacity: 0.5;
+  cursor: none;
+}
+
+.tool-parent {
+  position: relative;
+}
+
+.tooltip {
+  position: absolute;
+  bottom: 120%;
+  left: -120%;
+  width: 200px;
+  background-color: var(--gray-700);
+  border-radius: 5px;
+  padding: 0.5rem;
+}
+
+.tooltip::after {
+  content: '';
+  position: absolute;
+  top: 100%;
+  left: 50%;
+  border-width: 7px;
+  border-style: solid;
+  border-color: black transparent transparent transparent;
```

Comment:
Can we please use shorthands here?

Reply:
for width and style but color i am not able to use for all property.